models/five_fold_validation.py

```diff
@@ -0,0 +1,41 @@
+from logistic_regression import LogisticRegression
+import pandas as pd
+import numpy as np
+
+
+def validate_logistic_regression_for_wine_quality():
+    num_of_folds = 5
+    learning_rate= 0.000001
+    max_iterations = 100
+    df = pd.read_csv("../data/winequality/winequality-red.csv", sep=";")
+    df['classified']=[1 if x>=6 else 0 for x in df["quality"] ]
+    fold_size = int(round(df.shape[0]/num_of_folds))
+    for i in range(5):
+        x_test = df[['fixed acidity', 'volatile acidity', 'citric acid', 'residual sugar', 'chlorides', 'free sulfur dioxide', 'total sulfur dioxide', 'density', 'pH', 'sulphates', 'alcohol']][i*fold_size:fold_size+i*fold_size]
```

Comment:
I would assign df[['fixed acidity', 'volatile acidity', 'citric acid', 'residual sugar', 'chlorides', 'free sulfur dioxide', 'total sulfur dioxide', 'density', 'pH', 'sulphates', 'alcohol']]  to x_data, and df.classified = y_data

and then just use different slices from x_data and y_data. Instead of having to write the entire df thing again.

Reply:
I think we still have to slice and append. Because for training set we will take 2 different slices